common/test/acceptance/pages/lms/course_home.py

```diff
@@ -303,6 +303,13 @@ def _get_subsections_as_selenium_webelements(self):
         self._expand_all_outline_folds()
         return self.q(css=self._subsection_selector).results
 
+    def get_subsection_due_date(self, index=0):
+        """
+        Get the due date for the given index sub-section on the LMS outline.
+        """
+        results = self.q(css='div.details > span.subtitle > span.subtitle-name').results
+        return results[index].text
```

Comment:
Is there any possibility that results is None?

Reply:
Might be, if the course has no outline at all.

Reply:
Is this something that is worth handling?

Reply:
I just added this, and only my test is using it, so I don't think there is a need to deal with the special cases.